data/brands/shop/bag.json

```
@@ -132,6 +132,17 @@
         "shop": "bag"
       }
     },
+    {
+      "displayName": "Strandbags",
+      "id": "",
+      "locationSet": {"include": ["au"]},
+      "tags": {
+        "brand": "Strandbags",
+        "brand:wikidata": "Q1203Q111946652426",
```

Presumably this is an error

---

```suggestion
        "brand:wikidata": "Q111946652",
```
appears to be the correct one

---

That is correct, wikidata is Q111946652